FreeRADIUS/freeradius/ingest/parser.yml

```diff
@@ -0,0 +1,37 @@
+name: untitled-name
```

Comment:
Empty name here (where is this field used BTW?)

Reply:
Fixed